constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/ConstraintLayout.java

```diff
@@ -613,8 +613,7 @@ public void setId(int id) {
     // Measure widgets callbacks
     // -------------------------------------------------------------------------------------------
 
-
-    // -------------------------------------------------------------------------------------------
+// -------------------------------------------------------------------------------------------
```

Comment:
Doesn't look like this whitespace should have changed?
```suggestion
    // -------------------------------------------------------------------------------------------
```